sources/packages/api/src-sql/Student/Rollback-remove-sin-col.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS sin VARCHAR(10);
+
+-- Copy SIN from sin_validations to students.
+UPDATE
+    sims.students
+SET
+    sin = CAST(sims.sin_validations.sin AS VARCHAR(10))
```

Comment:
if the student has multiple sin rows in the `sin_validations`, when which one it will take, just for my info

Reply:
I was not so worried about picking a specific one. It was more about having a value to allow setting the `NOT NULL` constraint back. Maybe it would be good to add a where as below at least:
```sql
WHERE
    sims.sin_validations.user_id = students.user_id;
    AND sims.sin_validations.sin IS NOT NULL
```
Thoughts?

Reply:
mm...ya.. need not worry about it. but just wanted to know, if it's the first one or the last one. I think the current one should be fine since it will only happen when migration is reverted and which is not a scenario in PROD.

Reply:
The `WHERE` was not accurate. I changed it to use the `sin_validation_id ` already present on  `students.sin_validation_id = sims.sin_validations.id;`.